openvdb_points/tools/PointDataGrid.h

```diff
@@ -895,6 +905,25 @@ PointDataLeafNode<T, Log2Dim>::memUsage() const
     return BaseLeaf::memUsage() + mAttributeSet->memUsage();
 }
 
+
+////////////////////////////////////////
+
+
+template <typename PointDataTreeT>
+inline void
+makeDescriptorUnique(PointDataTreeT& tree)
+{
+    typename PointDataTreeT::LeafIter leafIter = tree.beginLeaf();
+    if (!leafIter)  return;
+
+    const AttributeSet::Descriptor& descriptor = leafIter->attributeSet().descriptor();
+    AttributeSet::Descriptor::Ptr newDescriptor(new AttributeSet::Descriptor(descriptor));
+    for (; leafIter; ++leafIter) {
+        leafIter->renameAttributes(descriptor, newDescriptor);
```

Comment:
Hmm looking at what renameAttributes is doing (replacing the descriptor), maybe the function call should be a bit more accurate on the PointDataGrid/AttributeSet? The free function in PointAttribute.h renameAttribute/s() makes sense, but maybe these can call through to something along the lines of "replaceDescriptor()", as the descriptor holds a lot more information that just attribute names.